components/expenses/ExpenseTypeRadioSelect.js

```diff
@@ -86,38 +76,42 @@ const ExpenseTypeOptionContainer = styled.label`
 `;
 
 const illustrations = {
-  [expenseTypes.INVOICE]: invoiceIllustration,
-  [expenseTypes.RECEIPT]: receiptIllustration,
-  [expenseTypes.GRANT]: grantAnimation,
+  [expenseTypes.INVOICE]: '/static/images/invoice-animation.gif',
+  [expenseTypes.RECEIPT]: '/static/images/receipt-animation.gif',
+  [expenseTypes.GRANT]: '/static/images/grant-animation.gif',
 };
 
 const staticIllustrations = {
-  [expenseTypes.INVOICE]: invoiceIllustrationStatic,
-  [expenseTypes.RECEIPT]: receiptIllustrationStatic,
-  [expenseTypes.GRANT]: grantIllustration,
+  [expenseTypes.INVOICE]: '/static/images/invoice-animation-static.jpg',
+  [expenseTypes.RECEIPT]: '/static/images/receipt-animation-static.jpg',
+  [expenseTypes.GRANT]: '/static/images/grant.gif',
 };
 
 const ExpenseTypeOption = ({ name, type, isChecked, onChange }) => {
   const { formatMessage } = useIntl();
-  const illustrationSrc = illustrations[type] || receiptIllustration;
-  const staticIllustrationSrc = staticIllustrations[type] || receiptIllustrationStatic;
+  const illustrationSrc = illustrations[type] || '/static/images/receipt-animation.gif';
+  const staticIllustrationSrc = staticIllustrations[type] || '/static/images/receipt-animation-static.jpg';
```

Comment:
:no_entry_sign: Here the animation should only happen on hover

![Peek 2022-09-16 10-26](https://user-images.githubusercontent.com/1556356/190571989-69a48607-2688-4723-bc85-916a8e038694.gif)

Reply:
@Betree : Ops, I believe we need to handle this a bit differently than I thought. I just made a fix for this. 👍🏽